src/index.tsx

```diff
@@ -33,3 +33,7 @@ export async function remove(key: string) {
 export async function clear() {
   await Cache.clear();
 }
+
+export const setItem = write;
+export const getItem = read;
+export const removeItem = remove;
```

Comment:
Not a big fan to have multiple export for the same function, but renaming current functions will be breaking.